.code-samples.meilisearch.yaml

```diff
@@ -8,7 +8,9 @@ get_one_index_1: |-
 list_all_indexes_1: |-
   client.indexes
 create_an_index_1: |-
-  client.create_index('movies', primary_key: 'movie_id')
+  client.create_index('movies')
```

Comment:
I dont think we need to remove the primary key https://github.com/meilisearch/documentation/blob/106625019b1f144036738d7db5491106fccecbbe/.code-samples.meilisearch.yaml#L12

Reply:
Also it is `id`! I think we have `movie_id` everywhere in our SDK's